app/components/DR/CasesStatistics.js

```diff
@@ -102,8 +103,20 @@ class CasesStatistics extends React.Component {
               styles.dateSubtitle,
               { alignSelf: 'center', textAlign: 'center' },
             ]}>
-            {this.getUpdateDate()}
+            {t('label.date_dashboard_label')}
           </Text>
+          <CalendarButton
+            style={{ backgroundColor: '#FFF' }}
+            date={
+              date
+                ? moment(date, 'YYYY-MM-DD').format('DD-MM-YYYY')
+                : moment(new Date()).format('DD-MM-YYYY')
+            }
+            onChange={date => {
+              this.getCases(moment(date, 'DD-MM-YYYY').format('YYYY-MM-DD'));
+            }}
+            minDate={moment(miniumDate)}
```

Comment:
I think moment is not necessary here

```suggestion
            minDate='2020-07-17'
```